src/robusta/cli/main.py

```diff
@@ -89,31 +86,6 @@ class HelmValues(BaseModel, extra=Extra.allow):
     runner: Dict = None
     rsa: RSAKeyPair = None
 
-    def set_pod_configs_for_small_clusters(self):
-        self.kubewatch = PodConfigs.gen_config(FORWARDER_CONFIG_FOR_SMALL_CLUSTERS)
```

Comment:
I think this was the only use of the PodConfigs class.
We can probably remove it